bcda/lambda/cclf/main.go

```diff
@@ -60,14 +60,56 @@ func cclfImportHandler(ctx context.Context, sqsEvent events.SQSEvent) (string, e
 			// importing the one file that was sent in the trigger.
 			filepath := fmt.Sprintf("%s/%s", e.S3.Bucket.Name, e.S3.Object.Key)
 			logger.Infof("Reading %s event for file %s", e.EventName, filepath)
-			return handleCclfImport(s3AssumeRoleArn, filepath)
+			if cclf.CheckIfAttributionCSVFile(e.S3.Object.Key) {
+				return handleCSVImport(s3AssumeRoleArn, filepath)
+			} else {
+				return handleCclfImport(s3AssumeRoleArn, filepath)
+			}
 		}
 	}
 
 	logger.Info("No ObjectCreated events found, skipping safely.")
 	return "", nil
 }
 
+func handleCSVImport(s3AssumeRoleArn, s3ImportPath string) (string, error) {
+	env := conf.GetEnv("ENV")
+	appName := conf.GetEnv("APP_NAME")
+	logger := configureLogger(env, appName)
+	logger = logger.WithFields(logrus.Fields{"import_filename": s3ImportPath})
+
+	importer := cclf.CSVImporter{
+		Logger: logger,
+		FileProcessor: &cclf.S3FileProcessor{
+			Handler: optout.S3FileHandler{
+				Logger:        logger,
+				Endpoint:      os.Getenv("LOCAL_STACK_ENDPOINT"),
+				AssumeRoleArn: s3AssumeRoleArn,
+			},
+		},
+	}
+
+	success, failure, skipped, err := importer.ImportCSV(s3AssumeRoleArn, s3ImportPath)
```

Comment:
It looks like you have left out the app and DB psuedo-code.  Is that because Lauren has it and is actively working on it?

Reply:
Yes, that's correct